README.md

```diff
@@ -22,16 +22,20 @@ pip install qa-eval
 Developed and maintained by [Graphwise](https://graphwise.ai/).
 For issues or feature requests, please open [a GitHub issue](https://github.com/Ontotext-AD/qa-eval/issues).
 
-## Final Answer Evaluation
+## Final Answer Evaluation Only
+
+To evaluate only final answers (system responses), you can clone this repository and run the code on the command line:
 
 1. Prepare an input TSV file with columns `Question`, `Reference answer` and `Actual answer`
 1. Modify file `answer_evaluation.py`, to set variable `DATA_FILE_PATH` to the path of the input TSV file
```

Comment:
If we're going in the direction of making this library a `cli`, users should be able to provide arguments to the commands, and not to be forced to change the source code I think

Reply:
Should we remove the CLI functionality for now?